docs/test_scripts.md

```diff
@@ -80,6 +80,18 @@ Will be turned into this:
 
 This is a useful feature for tests that are run against multiple instances of the same skill, where there are slight variations in the input or output.
 
+### Homophones
+It is now possible to specify homophone values for transcripts.
+
+This helps when the expected response is mis-understood - for example, the virtual device may come back with "okay" for "ok".
```

Comment:
I think it's the other way around: _the virtual device may come back with "ok" for "okay_ 
This is based on your later example where ok and OK would be replaced to "okay" in code.